src/main/java/mekanism/common/block/attribute/AttributeEnergy.java

```diff
@@ -1,20 +1,18 @@
 package mekanism.common.block.attribute;
 
-import mekanism.api.math.FloatingLong;
-import mekanism.api.math.FloatingLongSupplier;
+import java.util.function.LongSupplier;
 import net.minecraft.SharedConstants;
-import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 //TODO: Eventually we may want to make these suppliers be used more like suppliers in that:
 // if the config updates it doesn't require a server restart (or chunk reload to take effect
 public class AttributeEnergy implements Attribute {
 
-    private FloatingLongSupplier energyUsage = () -> FloatingLong.ZERO;
+    private LongSupplier energyUsage = () -> 0L;
     // 2 operations (20 secs) worth of ticks * usage
-    private FloatingLongSupplier energyStorage = () -> energyUsage.get().multiply(20 * SharedConstants.TICKS_PER_SECOND);
+    private LongSupplier energyStorage = () -> energyUsage.getAsLong() * (20 * SharedConstants.TICKS_PER_SECOND);
```

Comment:
We either should limit all the configs, or ensure this gets clamped out at `Long.MAX_VALUE` if it were to overflow.